crates/modelardb_server/src/parser.rs

```diff
@@ -708,7 +718,7 @@ mod tests {
     fn test_tokenize_and_parse_create_model_table() {
         let sql = "CREATE MODEL TABLE table_name(timestamp TIMESTAMP, field_one FIELD, field_two FIELD(10.5), tag TAG)";
         if let Statement::CreateTable { name, columns, .. } = tokenize_and_parse_sql(sql).unwrap() {
-            assert!(name == new_object_name("table_name"));
+            assert_eq!(name, new_object_name("table_name"));
```

Comment:
Sorry I did not think about it during the initial code review, but I think tests should be added that verify that the parser now rejects all table names that match keywords even if they are a different case. It should be simple to verify this by adding two tests: `test_tokenize_and_parse_create_model_table_with_lowercase_keyword_as_table_name()` and `test_tokenize_and_parse_create_model_table_with_uppercase_keyword_as_table_name()` that each loop over `ALL_KEYWORDS`.

Reply:
Something like this?

Reply:
Yes